xtask/src/ci/spec_links.rs

```diff
@@ -72,6 +72,16 @@ impl SpecLink {
     }
 }
 
+/// Whether an ID has duplicates.
+#[derive(Debug, Clone, Copy, Default, PartialEq, Eq)]
+enum HasDuplicates {
+    /// The ID has duplicates.
+    Yes,
+    /// The ID doesn't have duplicates.
+    #[default]
```

Comment:
The `Default` impl doesn't seem to be used, or am I missing sth.?

Reply:
It isn't, I thought I might need it :slightly_smiling_face:. I _could_ use it instead of using `No`.